src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -338,6 +337,7 @@ static std::vector<SenderThreadParams*>* buildSenderParamsCustom
     params->renderFormat     = renderFormatToString(renderFormat);
     params->fiwareCorrelator = fiwareCorrelator;
     params->extraHeaders     = headers;
+    params->registration     = false;
```

Comment:
Maybe a better name for this param (and closer to its semantic) would be `persistErrorStatusInCache` ("inverting" the current logic).

Just a suggestion...